core/services/ardupilot_manager/firmware_download/FirmwareDownload.py

```diff
@@ -241,21 +241,27 @@ def download(self, vehicle: Vehicle, platform: Platform, version: str = "") -> O
             logger.error(f"Specified version not found for this configuration ({vehicle} and {platform}).")
             return None
 
-        # Autodetect the latest stable version
```

Comment:
Commit message is wrong:
`- SITL validation is back in this commit, since the latest DEV fixes the non-zero return on --help call.`
This does not exist here.

Reply:
Fixed